nodesass.js

```diff
@@ -1,2 +1,4 @@
 var loader = require("@loader");
-module.exports = loader._nodeRequire("sass.js");
+// if we build your app with steal-tools, we are currently not in the steal-sass directory
+// we have to set the fully qualified path to the module
+module.exports = loader._nodeRequire(__dirname + "/node_modules/sass.js/dist/sass.sync.js");
```

Comment:
Yeah, I think `nodeRequire` is always going to be the user's require. However this change will only work with npm2, with npm3 sass.js could be somewhere else. We should use nodeRequire.resolve first, something like:

```js
var sassLocation = __dirname + "/node_modules/sass.js/dist/sass.sync.js";
try {
  sassLocation = loader._nodeRequire.resolve('sass.js/dist/sass.sync.js');
catch {}

module.exports = loader._nodeRequire(sassLocation);
```